src/client/src/Components/CardPreview/index.tsx

```diff
@@ -23,7 +23,7 @@ interface CardPreviewProps {
     onClick: () => void;
     isSelected: boolean;
     title: string;
-    values: { [key: string]: string | number | JSX.Element };
+    values: { [key: string]: string | number | JSX.Element | undefined };
```

Comment:
Is it possible to do 

`    values: { [key: string]?: string | number | JSX.Element};
`

Reply:
Not possible.